sphinx_rtd_theme/versions.html

```diff
@@ -1,34 +1,136 @@
-{% if READTHEDOCS %}
-{# Add rst-badge after rst-versions for small badge style. #}
-  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="{{ _('Versions') }}">
-    <span class="rst-current-version" data-toggle="rst-current-version">
-      <span class="fa fa-book"> Read the Docs</span>
-      v: {{ current_version }}
-      <span class="fa fa-caret-down"></span>
-    </span>
-    <div class="rst-other-versions">
+<!-- This style is injected ONLY on Read the Docs and it's missing. -->
+<style>
+ .rtd-current-item {
+   font-weight: bold;
+ }
+</style>
+
+<script type="text/javascript">
```

Comment:
Similar to above, we should no longer be using inline `script` for our code, it should go in our JS sources. Inline scripts and styles are one of the largest points for XSS, we should start a beta release from a solid base.

Reply:
dbefecb